pact/coin-contract/v2/coin.pact

```diff
@@ -326,6 +330,20 @@
       (credit account account-guard amount))
     )
 
+  (defun slash:string (account:string amount:decimal)
+    @doc "Internal function for the termination of coins. This function \
+         \cannot be used outside of the coin contract."
```

Comment:
Probably better to say something about it being protected by SLASH rather than "internal" and "cannot be used"